docs/roadmap.rst

```diff
@@ -0,0 +1,123 @@
+.. _roadmap:
+
+Roadmap
+=======
+
+The long-term vision for :py:mod:`pysat` is to make the package suitable for
+working with any combination of data. As :py:mod:`pysat` is intended to support
+the development of highly robust and verifiable scientific analysis and
+processing packages, :py:mod:`pysat` must produce each of its features with high
+quality unit tests and documentation.
+
+This document provides a broad and long term vision of :py:mod:`pysat`. Specific
+tasks associated with this roadmap may be found within the posted
+`Issues <https://github.com/pysat/pysat/issues>`_ and
+`Projects <https://github.com/pysat/pysat/projects>`_.
+
+An item being on the roadmap does not necessarily mean that it will happen.
+During the implementation or testing periods we may discover issues that limit
+the feature.
+
+Generality
+----------
```

Comment:
```suggestion
```

Recommend removing this header.  The word is not used correctly (from what I could see looking up the definition) and the text goes well under the top level header.